src/InputField/inputField.css

```diff
@@ -64,23 +60,24 @@
 .fakeHovered
```

Comment:
`forceHover`/`fakeHovered` styling no longer needed as we don’t have a hover effect anymore (let’s keep the prop but remove the styling)